app/models/concerns/digital_object/xml_datastream_rendering.rb

```diff
@@ -156,6 +156,9 @@ def value_for_field_name(field_name, df_data)
         created_at
       elsif field_name == '$updated_at'
         updated_at
+      elsif field_name == '$doi'
+        # slice off the "doi:" label
+        doi[4..-1] unless doi.nil?
```

Comment:
Would you mind changing line 161 to `doi.gsub(/^doi:/, '') unless doi.nil?`

That'll be more future-compatible when we remove the "doi:" prefix from values.